### ISSUE TITLE
Sync Failure

### ISSUE THREAD

**Issue by tuoggy** [stefan-niedermann/nextcloud-notes#1153]
Having an issue syncing with the most recently updated version on Pixel 3.

I'm using Notes v4.0.4 on NC Hub v21.0.1

Here are the details of the error

```
App Version: 3.3.0
App Version Code: 3003000
App Flavor: fdroid

Files App Version Code: 30150190

---

OS Version: 4.9.237-gbd4db4f9f99f-ab7079298(7119741)
OS API Level: 30
Device: blueline
Manufacturer: Google
Model (and Product): Pixel 3 (blueline)

---

b.b.a.a.e.n: b.b.a.a.c.c
	at b.b.a.a.d.c.i(Unknown Source:33)
	at b.b.a.a.d.e.c(Unknown Source:6)
	at it.niedermann.owncloud.notes.persistence.p.e(Unknown Source:8)
	at it.niedermann.owncloud.notes.persistence.i.a(Unknown Source:124)
	at it.niedermann.owncloud.notes.main.MainActivity.C0(Unknown Source:71)
	at it.niedermann.owncloud.notes.main.MainActivity.D0(Unknown Source:0)
	at it.niedermann.owncloud.notes.main.l.run(Unknown Source:4)
	at java.lang.Thread.run(Thread.java:923)

```

**Comment by Minnietheandroid** [comment]
Same here, reinstalled last version and ok again.

```
App Version: 3.3.0
App Version Code: 3003000
App Flavor: fdroid

Files App Version Code: 30150190

---

OS Version: 4.4.210-lineage-g53c17eb(7ec7e97251)
OS API Level: 29
Device: taimen
Manufacturer: Google
Model (and Product): Pixel 2 XL (taimen)

---

b.b.a.a.e.n: b.b.a.a.c.c
	at b.b.a.a.d.c.i(Unknown Source:33)
	at b.b.a.a.d.e.c(Unknown Source:6)
	at it.niedermann.owncloud.notes.persistence.p.e(Unknown Source:8)
	at it.niedermann.owncloud.notes.persistence.i.a(Unknown Source:124)
	at it.niedermann.owncloud.notes.main.MainActivity.C0(Unknown Source:71)
	at it.niedermann.owncloud.notes.main.MainActivity.D0(Unknown Source:0)
	at it.niedermann.owncloud.notes.main.l.run(Unknown Source:4)
	at java.lang.Thread.run(Thread.java:919)

```

**Comment by InAUGral** [comment]
I have the same error. Although I already tried to reinstall the app with no success. I have installed both latest and the last version which I know was working fine  before I updated. Also I am installing it from the F-Droid version. Looks like the errors above are from the same version.

```
App Version: 3.3.0
App Version Code: 3003000
App Flavor: fdroid

Files App Version Code: 30150190

---

OS Version: 3.18.91-16526088(J730GDXSDCUC1)
OS API Level: 28
Device: j7y17lte
Manufacturer: samsung
Model (and Product): SM-J730G (j7y17ltedx)

---

b.b.a.a.e.n: b.b.a.a.c.c
	at b.b.a.a.d.c.i(Unknown Source:33)
	at b.b.a.a.d.e.c(Unknown Source:6)
	at it.niedermann.owncloud.notes.persistence.p.e(Unknown Source:8)
	at it.niedermann.owncloud.notes.persistence.i.a(Unknown Source:124)
	at it.niedermann.owncloud.notes.main.MainActivity.C0(Unknown Source:71)
	at it.niedermann.owncloud.notes.main.MainActivity.D0(Unknown Source:0)
	at it.niedermann.owncloud.notes.main.l.run(Unknown Source:4)
	at java.lang.Thread.run(Thread.java:764)

```

**Comment by tuoggy** [comment]
Installing the previous version (3.2.5) works for me.

**Comment by InAUGral** [comment]
> 
> Installing the previous version (3.2.5) works for me.
> […](#)


Oh yep I installed 3.2.5 and its working again. Must have had that version on my phone previously and not realized there was a version in between.

**Comment by stefan-niedermann** [comment]
This bug is solved in `3.3.1`. Please see the pinned issues next time.
